src/Kunstmaan/TranslatorBundle/Service/Command/Importer/ImportCommandHandler.php

```diff
@@ -90,6 +93,10 @@ private function importAllBundlesTranslationFiles(ImportCommand $importCommand)
         $bundles = array_map('strtolower', array_keys($this->kernel->getBundles()));
         $imported = 0;
 
+        if (Kernel::VERSION_ID >= 40000) {
```

Comment:
I would put this check in the `importBundleTranslationFiles`method under the call `importAllBundlesTranslationFiles` because `importSf4TranslationFiles` technically has nothing to do with `bundles` and more relates to the `all` check that is being done.